django/text_extractor/utils.py

```diff
@@ -285,3 +285,12 @@ def create_searchable_pdf(input_file, add_header):
         output.add_page(new_pdf_page.pages[0])
 
     return output, all_text
+
+
+def shorten_input_name(input_name, max_length=35):
+    base_name, file_extension = os.path.splitext(input_name)
+    if len(base_name) > max_length:
+        unique_id = str(uuid.uuid4())
+        shortened_base_name = base_name[:10] + "_" + unique_id
```

Comment:
You can just use the UUID, no need for the base name (although... it may be nice for debugging. Your call).